tests/e2e/taqueria-plugin-taquito.spec.ts

```diff
@@ -0,0 +1,346 @@
+            // It is applicable only for auto-tests that run very fast
+            // Uses retry mechanism to avoid test to fail
+            await waitForExpect(() => {
+                // 2. Run taq deploy on a network described in "test" environment
+                const stdoutDeploy = execSync(`taq deploy -e ${environment}`, {cwd: `./${taqueriaProjectPath}`}).toString().trim().split(/\r?\n/);
+
+                // 3. Verify that contracts have been originated on the network
+                expect(stdoutDeploy[3]).toContain("hello-tacos-one.tz");
+                expect(stdoutDeploy[3]).toContain("hangzhounet");
+                const [,,contractOneHash] = stdoutDeploy[3].split("│");
+                smartContractOneHash = contractOneHash.trim();
+                expect(smartContractOneHash).toMatch(contractRegex);
+                expect(stdoutDeploy[5]).toContain("hello-tacos-two.tz");
+                expect(stdoutDeploy[5]).toContain("hangzhounet");
+                const [,,contractTwoHash] = stdoutDeploy[5].split("│");
+                smartContractTwoHash = contractTwoHash.trim();
+                expect(contractTwoHash.trim()).toMatch(contractRegex);
+            });
+
+            // 4. Verify that contracts have been originated to the network
+            await waitForExpect(async () => {
+                const contractOne = await tezos.contract.at(smartContractOneHash);
+                expect(contractOne.address).toBe(smartContractOneHash);
+                const contractTwo = await tezos.contract.at(smartContractTwoHash);
+                expect(contractTwo.address).toBe(smartContractTwoHash);
+            });
+
+        } catch(error) {
+            throw new Error (`error: ${error}`);
+        }
+
+    });
+
+    // TODO: Currently there is no specific message for this type of error
+    // Issue associated https://github.com/ecadlabs/taqueria/issues/313
+    test.skip('Verify that taqueria taquito plugin will show proper error when environment does not exists', async () => {
+        try {
+            // Environment test does not exist on default config.json
+            environment = "test"
+
+            // 1. Copy config.json and two michelson contracts from data folder to artifacts folder under taqueria project
+            execSync(`cp e2e/data/hello-tacos.tz ${taqueriaProjectPath}/artifacts/hello-tacos.tz`);
+
+            // 2. Run taq deploy on a network described in "test" environment
+            const stdoutDeploy = execSync(`taq deploy -e ${environment}`, {cwd: `./${taqueriaProjectPath}`}).toString().trim();
+
+            console.log(stdoutDeploy);
+
+            // 3. Verify that proper error displays in the console
+            // expect(stdoutDeploy).toContain("E_INVALID_PLUGIN_RESPONSE");
+
+        } catch(error) {
+            throw new Error (`error: ${error}`);
+        }
+
+    });
+
+    // TODO: Currently there is no specific message for this type of error
```

Comment:
Please see comment above. We can probably create one issue for enabling any of these skipped tests that are dependent on this problem being resolved.

Reply:
Fixed. Issue to investigate and re-enable tests - https://github.com/ecadlabs/taqueria/issues/377